### ISSUE TITLE
The results can't be reproduced

### ISSUE THREAD

**Issue by xhanxu** [tangjiapeng/DiffuScene#23]
I use the pretrained model to generate 1000 images (256 x 256) like this:
![rendered_scene_notexture_256](https://github.com/tangjiapeng/DiffuScene/assets/77310777/4882efec-666c-4f43-bd39-2e27a38e4124)

I also make sure the 4041 GT images in 3D-FRONT are like this either. 
When I use the **4041 GT images** and **1000 predicted images** to evaluate the **pretrained model**, I get `fid=26.23`, `kid=0.0046`, `SCA=0.66`(improved precision?). 
According to the paper, it should be **`fid=17.21`**, **`kid=0.0007 (0.70 x 0.001)`**, **`SCA=0.5215`**, which is much better than my results, right? 

Please let me know if there is anything wrong. 

Plus, is there any chance that the CKL code will be released?

Thanks again for your great work!

**Comment by tangjiapeng** [comment]
did you try my pretrained checkpoint for debugging?

**Comment by xhanxu** [comment]
Thank you for response!

Yes, I use the pretrained weights that you released (bedroom), and I got `FID=25.55, KID=3.96, SCA=66.14`, which is still far away from `FID=17.21, KID=0.70, SCA=52.15`.

**Comment by tangjiapeng** [comment]
We use the train/val scenes as the reference set. And not quite sure if you use the same rendering setting for reference scenes. 
You can have a look at the top-down rendering images of both generated and reference scenes.

**Comment by xhanxu** [comment]
Got it! I will check those settings again.

Thanks for your help!

**Comment by Yukisha04** [comment]
Hello Can you guide me how to implement the pretrained model on the pretrained dataset? 
Thank you

**Comment by Sun-XH** [comment]
Hi @xhanxu 

I met the same issue as you. I used the pretrained model and got similar results as yours. Have you found out why the results are far from what the paper claimed? Looking forward to your reply.

Thanks

**Comment by Lizhenghe-Chen** [comment]
> I also make sure the 4041 GT images in 3D-FRONT

where to get these GT images in 3D-FRONT? I cant find them in the google drive datasets...
@xhanxu @ @

**Comment by tangjiapeng** [comment]
can you access to this gdrive link ?

https://drive.google.com/drive/folders/1EhvyNCAWWto6vMt0vXWMKBoSdYR_9pC2?usp=sharing

**Comment by Lizhenghe-Chen** [comment]
> can you access to this gdrive link ?
> 
> https://drive.google.com/drive/folders/1EhvyNCAWWto6vMt0vXWMKBoSdYR_9pC2?usp=sharing

I can access the link, but I'm unsure where the Ground Truth (GT) images are located. It seems that `/cluster/balrog/jtang/3d_front_processed/bedrooms_objfeats_32_64` is the intended path, one of them look like this:

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/74a8be98-e8b2-45c7-9bf1-f042e99c258b" />

 but they are not the same as the generated image(Semantic-Segmented Depth Images), so the evalutaion result are tolltaly wrong:

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/a409ca85-dbb4-42dc-ad2f-d79257636d6d" />

**Comment by tangjiapeng** [comment]
I have added the ground-truth scene renderings with semantic labels. You can find them under the suffix *_notexture_nofloor_whiteground:
https://drive.google.com/file/d/1gGODp3uMhgV4X45u3akdkQAdJw2pv_FG/view?usp=sharing

Our evaluation protocols and results are consistent with the figures presented in the paper. 

Please refrain from concluding that "the evaluation results are totally wrong" until you have thoroughly investigated any potential issues.

**Comment by Lizhenghe-Chen** [comment]
> I have added the ground-truth scene renderings with semantic labels. You can find them under the suffix *_notexture_nofloor_whiteground: https://drive.google.com/file/d/1gGODp3uMhgV4X45u3akdkQAdJw2pv_FG/view?usp=sharing
> 
> Our evaluation protocols and results are consistent with the figures presented in the paper.
> 
> Please refrain from concluding that "the evaluation results are totally wrong" until you have thoroughly investigated any potential issues.

Many thanks for the update, and sorry for the confusion. I meant that the result is definitely wrong for me because I used the wrong GT datasets, not that your research result is wrong.🤣
